deploy/openshift/templates/monitoring/grafana-dashboards.yaml

```diff
@@ -3721,7 +3727,7 @@ data:
               "value": "null"
             }
           ],
-          "valueName": "current"
+          "valueName": "avg"
```

Comment:
@tomgeorge so, currently the `Number of Users` / `Number of Workspaces` will show the average for the given timeframe? Not sure if it makes sense... I would opt for Current number of users & workspaces

Reply:
Good catch, changed back to current.